### PR TITLE
Switch from using Travis to GitHub Actions for release

### PR DESCRIPTION
**Pull request by rnorth**
Undoing #2279 - need to follow up on https://github.community/t/read-timeout-issues-uploading-large-number-of-files-to-bintray-with-github-actions/17243/3#M5104

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '1.8'
+      - name: Clear existing docker image cache
+        run: docker image prune -af
+      - id: gradle-release
+        run: ./gradlew -Pversion=$GITHUB_REF release --scan --no-daemon -i
```
Comment (bsideup) [inline .github/workflows/release.yml:24]:
```suggestion
        run: ./gradlew -Pversion="${{github.event.release.tag_name}}" release --scan --no-daemon -i
```

File: README.md [hunk 2]
```diff
@@ -2,8 +2,6 @@
 
 > Testcontainers is a Java library that supports JUnit tests, providing lightweight, throwaway instances of common databases, Selenium web browsers, or anything else that can run in a Docker container.
 
-[![Build Status](https://travis-ci.org/testcontainers/testcontainers-java.svg?branch=master)](https://travis-ci.org/testcontainers/testcontainers-java)[ ![Download](https://api.bintray.com/packages/testcontainers/releases/testcontainers/images/download.svg) ](https://bintray.com/testcontainers/releases/testcontainers/_latestVersion)
```
Comment (rnorth) [inline README.md:5]: Just removing without replacing, as I doubt anybody will miss it.

File: build.gradle [hunk 3]
```diff
@@ -60,8 +57,6 @@ subprojects {
                 }
             }
         }
```
Comment (rnorth) [inline build.gradle:59]: It feels like this `publishing` block should probably be consolidated into `publishing.gradle`. Will do that, checking that there's not some weird side-effect in the set of generated artifacts.

Comment (rnorth) [reply build.gradle]: Done

**Review by kiview — APPROVED** [review]

**Review by bsideup — APPROVED** [review]

**Pull request merged**